mangle_data.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
I have no strong objection to us using python here; I wonder (if we're going to keep using it); if we'd benefit from throwing a light sprinkling of all the usual tooling here: some linting, maybe even some super light e2e test ?

What do you think?

Reply:
I think I would prefer not having to introduce dependency management for Python here, but having a small test case written as a shell script might be a nice addtion, yes. I'll see what I can come up with.